io.github.KangLin.RabbitRemoteControl.json

```diff
@@ -0,0 +1,241 @@
+        {
+            "name": "FreeRDP",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/FreeRDP/FreeRDP/archive/refs/tags/3.14.0.tar.gz",
+                    "sha256": "a7a979382703c2bc18aa9b22cfa12d7a441b6927ab45ac62ef8353786bc92241"
+                }
+            ],
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "config-opts": [
+                "-DCMAKE_VERBOSE_MAKEFILE=OFF",
+                "-DCMAKE_BUILD_TYPE:STRING=Release",
+                "-DCHANNEL_TSMF:BOOL=OFF",
+                "-DBUILD_TESTING:BOOL=OFF",
+                "-DWITH_MANPAGES:BOOL=OFF",
+                "-DWITH_WAYLAND:BOOL=OFF",
+                "-DWITH_GSSAPI:BOOL=OFF",
+                "-DWITH_SAMPLE:BOOL=OFF",
+                "-DWITH_OSS:BOOL=OFF",
+                "-DCHANNEL_URBDRC:BOOL=OFF",
+                "-DWITH_CLIENT_SDL=OFF",
+                "-DWITH_CLIENT=OFF",
+                "-DWITH_KRB5=ON"
+            ],
+            "cleanup": [
+                "/bin",
+                "/share"
+            ]
+        },
+        {
+            "name": "libvncserver",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/LibVNC/libvncserver/archive/refs/tags/LibVNCServer-0.9.15.tar.gz",
+                    "sha256": "62352c7795e231dfce044beb96156065a05a05c974e5de9e023d688d8ff675d7"
+                }
+            ],
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "config-opts": [
+                "-DWITH_OPENSSL=ON",
+                "-DWITH_GCRYPT=OFF",
+                "-DWITH_EXAMPLES=OFF",
+                "-DWITH_TESTS=OFF"
+            ],
+            "cleanup": [
+                "/bin/*",
+                "/lib/libvncserver.so*"
+            ]
+        },
+        {
+            "name": "cmark",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/commonmark/cmark/archive/refs/tags/0.31.1.tar.gz",
+                    "sha256": "3da93db5469c30588cfeb283d9d62edfc6ded9eb0edc10a4f5bbfb7d722ea802"
+                }
+            ],
+            "buildsystem": "cmake",
+            "builddir": true,
+            "config-opts": [
+                "-DBUILD_TESTING=OFF"
+            ],
+            "cleanup": [
+                "/bin/*"
+            ]
+        },
+        {
+            "name": "RabbitCommon",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/KangLin/RabbitCommon.git",
+                    "branch": "master",
+                    "commit": "f2ccefec47d948191b6060e9b89f5e82bf1307fc"
+                }
+            ],
+            "buildsystem": "simple",
+            "build-commands": [
+                "cp -R /run/build/RabbitCommon /var/tmp/"
```

Comment:
i still don't understand /var/tmp here? You can use /run/build/$module_name or copy it somewhere to /app and cleanup it later.